### PR TITLE
Hardcoded version of ES client python library

### PR DESCRIPTION
**Pull request by doublebyte1**
Specified version of the python client of ES, to avoid breaking changes from version [v8.0.0](https://github.com/elastic/elasticsearch-py/releases/tag/v8.0.0).

### CODE REVIEW

**Comment by tomkralidis** [comment]
Thanks @doublebyte1 !  Should we pin the ES requirement (perhaps to `<8`) in the following additional areas?

- https://github.com/geopython/pygeoapi/blob/master/requirements-provider.txt#L1
- https://github.com/geopython/pygeoapi/blob/master/debian/control#L26
- https://github.com/geopython/pygeoapi/blob/master/requirements-provider.txt#L13 (cc @francbartoli)

**Comment by doublebyte1** [comment]
@tomkralidis Please check my latest commit : https://github.com/geopython/pygeoapi/pull/862/commits/d93672ca9b37d06becc32971657ced3f1415e5f5

File: debian/control [hunk 1]
```diff
@@ -23,7 +23,7 @@ Depends: ${python3:Depends},
          python3-unicodecsv,
          python3-yaml,
          ${misc:Depends}
-Suggests: python3-babel, python3-elasticsearch, python3-fiona, python3-geojson, python3-pydantic, python3-pygeometa, python3-pyproj, python3-rasterio
+Suggests: python3-babel, python3-elasticsearch<8.0.0, python3-fiona, python3-geojson, python3-pydantic, python3-pygeometa, python3-pyproj, python3-rasterio
```
Comment (tomkralidis) [inline debian/control:26]: I think this should be `python3-elasticsearch (<8)` ?

File: requirements-provider.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-elasticsearch==7.1.0
+elasticsearch<8.0.0
```
Comment (tomkralidis) [inline requirements-provider.txt:1]: `elasticsearch<8`

File: requirements-provider.txt [hunk 3]
```diff
@@ -10,4 +10,5 @@ pymongo==3.10.1
 scipy
 xarray
 zarr
-elasticsearch-dsl
+elasticsearch-dsl<8.0.0
```
Comment (tomkralidis) [inline requirements-provider.txt:13]: `elasticsearch<8`

**Review by tomkralidis — CHANGES_REQUESTED** [review]

**Review by francbartoli — APPROVED** [review]

**Pull request merged**